distribution/changelog.txt

```diff
@@ -1,4 +1,4 @@
-0.1.3 (in development)
+﻿0.1.3 (in development)
```

Comment:
It looks like your editor added a byte order mark (BOM) here. Could you remove it?

Reply:
FWIW Notepad does this by default, Notepad++ and Visual Studio do not.

Reply:
Thank you. Notepad decided to add it. I've removed it with VIM.